### ISSUE TITLE
List view for diary entries

### ISSUE THREAD

**Issue by hspagnolo** [samuelmeuli/mini-diary#68]
Hello,

First, thanks for this great application. Would it be possible to have an option to display (like the search results) all diaries that have been entered? 

Well, as I become older and older, I do not necessarily remind what I have entered and when, so this can helps. Else we have to display month per month and click on every highlighted date to display the diary.

Thanks and regards
Henri

**Comment by DrLeucine** [comment]
This would be very appreciated. It helps to go back and revisit old entries.

**Comment by oneduality** [comment]
Agreed.. this is the one feature that is holding me back from heavier use.. 

I have a bad memory due to a condition I have and so I've decided to start documenting my days a bit so I can go back if I need to..  a list view will be important as entries grow..  search only helps if I know what I'm searching for.

**Comment by putnamcity** [comment]
I agree.  Would be a great feature.
